webpack/assets/javascripts/react_app/redux/actions/common/getTableItemsAction.js

```diff
@@ -0,0 +1,21 @@
+import URI from 'urijs';
```

Comment:
I would prefer to put this file next to the table code.

Could you move it to `components/common/table/actions/.`?

Reply:
bump?